lib/tasks/data_fixes/adjust_subjects_and_creators.rake

```diff
@@ -0,0 +1,61 @@
+namespace :scihist do
+  namespace :data_fixes do
+    desc "Subject and creator changes to match name authorities, per https://github.com/sciencehistory/scihist_digicoll/issues/1432"
+    task :adjust_subjects_and_creators => :environment do
+
+      class SubjectCreatorAdjuster
```

Comment:
Not sure whether just dumping a helper class inside the task scope is such a great plan.

Reply:
Hm... looks like it *is* just scoped locally. I kind of don't want it in `/.lib` : in its current form it has no use outside this one task. But also, referring to the class in the task before it's declared results in a `NameError`.
Don't want to agonize over this too much, but other ideas welcome!

Reply:
All makes sense. I moved stuff around a bit and put the helper class inside the namespace, right after the task.
That way, the task can instantiate the helper class, and the helper class is invisible outside of the namespace.

Reply:
OK, in my tests the helper class is *not* invisible outside the namespace. For me, it's totally visible outside the namespace, it's global. 

Maybe I'm wrong? What did you look at to conclude it is invisible outside the namespace or not globally referenceable?

But of course if we put it in `./lib` it would be global too. I think it's fine, I'd just give it a very specific name unlikely to conflict. 

And personally I'd just put it at the bottom of the file, so it doesn't *look* like it's going to be non-global/namespaced... if i'm right that it's really not.

Reply:
This blog post warns you about defining isolated methods with `def` (something I used to do too, mistakenly!), but the logic applies to classes too. 

https://kevinjalbert.com/defined_methods-in-rake-tasks-you-re-gonna-have-a-bad-time/

 And my own testing just now seemed to reveal that it's global:

```ruby
namespace :test do
  task :test do
    Foo.new.call("inside")
  end

  class Foo
    def call(str)
      puts "called #{str}"
    end
  end
end

# Do we have it? if it were invisible here or not reference-able as "foo", we'd get
# an error on loading this file, but instead it manages to call the method and print
# "called outside" just fine
Foo.new.call("outside")


```